survey-service/src/main/java/pl/hycom/surveyservice/dto/DtoPage.java

```diff
@@ -0,0 +1,55 @@
+package pl.hycom.surveyservice.dto;
+
+import java.util.List;
+
+public class DtoPage {
```

Comment:
PageDTO lepiej brzmi, częściej takie nazewnictwo się spotyka w projektów, najpierw nazwa modelowa potem nazwijmy to funkcyjna, ale to o nazewnictwie pogadamy więcej w czwartek.

Reply:
ok, poprawiłem, na PageDTO i SurveyDTO